docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.links.scss

```diff
@@ -71,7 +70,7 @@
   }
 
   @include hb-traditional {
-    @include hb-link-background-image($icon, $icon-width, $hb-traditional-default, $ht-traditional-pairings);
+    background-image: none;
```

Comment:
will you place lines 63-65 inside the `hb-themes(('colorful', 'airy'))` statement?

Reply:
also some of the hover / focus stuff that references background image below

Reply:
I changed the order a little bit. I prefer to see the common styles, then colorful/airy, and finally traditional. 🤷

Reply:
me too!! looks great